Dockerfile

```
@@ -25,9 +23,7 @@ COPY . .
 ENV GOOS=linux
 ENV GOARCH=amd64
 ENV CGO_ENABLED=0
-# Run full test suite and build the static binary
-RUN make test TAGS=integration \
-    && make build GOBIN=/go/bin
+RUN make build GOBIN=/go/bin
```

Do we need to set `GOBIN` here, should it depend on `GOPATH`?

```suggestion
RUN make build GOBIN=${GOPATH}/bin
```

---

Yeah, I think you are correct. I validated locally and it is equivalent when not set. I think this is a hold over from when this was built with a different container.